src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -169,22 +168,7 @@ protected override void Validate()
         {
             if (CommonParameters.MtlsCertificate != null)
             {
-                string authorityUri = ServiceBundle.Config.Authority.AuthorityInfo.CanonicalAuthority.AbsoluteUri;
-
-                if (ServiceBundle.Config.Authority.AuthorityInfo.AuthorityType != AuthorityType.Aad && 
-                    ServiceBundle.Config.Authority.AuthorityInfo.AuthorityType != AuthorityType.Dsts)
```

Comment:
dSTS doesn't have the same rules.

Reply:
@bgavrilMS - Can you expand? This is the existing code. The issue #5093 is to disallow ``/organizations``.

Reply:
@bgavrilMS @gladjohn - Is there an action item here with respect to this PR?

Reply:
@bgavrilMS - Resolving as we agreed to remove the check.